ADOPTERS.md

```diff
@@ -40,6 +40,20 @@ Details (optional):
 - 25000 samples/s
 - 1.2M active series
 
+## Giant Swarm
+
+https://www.giantswarm.io/
+
+Environments: AWS, Azure, Bare Metal
+
+Uses [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus): Yes (with additional tight Giant Swarm integrations)
```

Comment:
Nice, we are looking forward if that is something you can bring into kube-prometheus or needs to be on Giant Swarm level?

Reply:
Our fork has minimal changes and are mostly on the Giant Swarm level. We tend to provide things we need/want upstream directly. 
For example, we disable app armor by default in the values file, add some files for our UI and we also retag the upstream images to be able to use them in China (we push them to a registry there to avoid timeouts during pulls). 

With the recent PR to be able to change the default image used by the prometheus operator, this part will be easier to change :)

You can find our version here https://github.com/giantswarm/prometheus-operator-app